package.json

```diff
@@ -20,7 +20,7 @@
         "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
```

Comment:
can we prerelease this in a dev branch, and test out the implications. then we can merge to master

Reply:
Let me see if I can build and install locally.